src/settings/JobProfiles/JobProfiles.js

```diff
@@ -26,6 +26,85 @@ import { JobProfilesForm } from './JobProfilesForm';
 const INITIAL_RESULT_COUNT = 5000;
 const RESULT_COUNT_INCREMENT = 5000;
 
+export const jobProfilesShape = {
+  INITIAL_RESULT_COUNT,
+  RESULT_COUNT_INCREMENT,
+  manifest: {
+    initializedFilterConfig: { initialValue: false },
+    query: { initialValue: {} },
+    resultCount: { initialValue: INITIAL_RESULT_COUNT },
+    records: {
+      type: 'okapi',
+      perRequest: RESULT_COUNT_INCREMENT,
+      records: ENTITY_KEYS.JOB_PROFILES,
+      recordsRequired: '%{resultCount}',
+      path: 'data-import-profiles/jobProfiles',
+      clientGeneratePk: false,
+      throwErrors: true,
+      GET: {
+        params: {
+          query: makeQueryFunction(
+            'cql.allRecords=1',
+            '(name="%{query.query}*" OR tags.tagList="%{query.query}*")',
+            {
+              name: 'name',
+              tags: 'tags.tagList',
+              updated: 'metadata.updatedDate',
+              updatedBy: 'userInfo.firstName userInfo.lastName userInfo.userName',
+            },
+            [],
+          ),
+        },
+        staticFallback: { params: {} },
+      },
+    },
+  },
+  visibleColumns: [
+    'name',
+    'description',
+    'tags',
+    'updated',
+    'updatedBy',
+  ],
+  columnWidths: {
+    isChecked: '35px',
+    name: '300px',
```

Comment:
Are hardcoded widths still needed? MCL already has already a mechanism for calculating widths automatically, so `columnWidths` usage can be reduced. The same is true for other profiles.